src/google-maps/google-map/google-map.ts

```diff
@@ -444,6 +444,14 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
     this._assertInitialized();
     return this.googleMap.getZoom();
   }
+  
+  /**
+   * See
+   * https://developers.google.com/maps/documentation/javascript/reference/map#Map.setZoom
+   */
+  setZoom(zoom: number): number {
+    this.googleMap.setZoom(zoom);
```

Comment:
`this._assertInitialized();` must be called before.

Reply:
@bastienlemaitre Thx, but not sur I understand.

Should I or will you make that changes ?